resources/etc/init.d/openhab2

```diff
@@ -12,30 +12,60 @@
 # Author: openHAB <openhab@openhab.org>
 #
 
+# Read configuration variable file if it is present
+if [ -r /etc/default/openhab2 ]; then
+  . /etc/default/openhab2
+fi
+
+if [ -r /etc/default/openhab2_dirs ]; then
+  . /etc/default/openhab2_dirs
+fi
+
+
+if [ -n "${EXTRA_JAVA_OPTS}" ]; then export EXTRA_JAVA_OPTS; fi
+
+# Read and set openHAB variables set in /etc/default/ scripts
+if [ -n "${OPENHAB_HTTP_PORT}" ]; then export OPENHAB_HTTP_PORT; fi
+if [ -n "${OPENHAB_HTTPS_PORT}" ]; then export OPENHAB_HTTPS_PORT; fi
+if [ -n "${OPENHAB_HOME}" ]; then export OPENHAB_HOME; fi
+if [ -n "${OPENHAB_CONF}" ]; then export OPENHAB_CONF; fi
+if [ -n "${OPENHAB_RUNTIME}" ]; then export OPENHAB_RUNTIME; fi
+if [ -n "${OPENHAB_USERDATA}" ]; then export OPENHAB_USERDATA; fi
+if [ -n "${OPENHAB_LOGDIR}" ]; then export OPENHAB_LOGDIR; fi
```

Comment:
Excuse me if I'm overlooking something. :speak_no_evil:
Example: Down in `dostart` you are using `"${OPENHAB_HOME}"`. What happens if the user would damage, empty or delete the files in /etc/default? The variable is unset and will not be set here. Down in dostart an empty string is inserted.

Defaults are not necessarily what I'm asking for but there should be some kind of sanity check to see if the variables are set, if not *do something* (defaults, fail, start crying).
Am I wrong?